src/components/modules/MainPage/ShortCard/ShortCard.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import {
+  Container,
+  Content,
+  ImageHolder,
+  DataHolder,
+  Image,
+  Description,
+  FooterContent,
+  ButtonSty,
+  TextBold,
+  SectionDiv,
+  HeadLine
+} from "./StyledShortCard";
+import CardTitle from "../../../common/Card/CardTitle";
+
+/**
+ * Props:
+ *  - title:
+ *  - description:
+ *  - imageSrc:
+ *  - size:
+ *  - href:
+ *  - onClick:
+ *  - attachments:
+ *  - textAlignment: changes text-alignment if passed otherwise it defaults to left text-alignment
+ *  - imageHolderBackgroundColor: changes bgColor if passed otherwise it defaults to black bgColor
+ *  - cardFlexDirection: changes flex-direction if existed otherwise delete flex-direction
+ */
+class ShortCard extends React.Component {
```

Comment:
I would prefer to be consistent with the rest of the codebase, would be able to turn this into a functional component?

Reply:
<img width="1098" alt="Screen Shot 2022-03-26 at 11 32 44 PM" src="https://user-images.githubusercontent.com/58893691/160269723-3f30e0e7-ac43-4856-9031-4823065d5e6a.png">

Reply:
![Screen Shot 2022-03-26 at 11 33 03 PM](https://user-images.githubusercontent.com/58893691/160269735-78a4c485-a319-4345-b29f-2a2b8c40e33d.png)

Reply:
I try to use a function component,  the cardDatas base has been transferred to the ShortCard, but why **cardDatas.time** **cardDatas.content****cardDatas.position** undefined ?

Reply:
all right , never mind, I am missing  {}, it should be    **const {cardDatas} = props;**

Reply:
It has been change to a function component, done !